composer.json

```diff
@@ -14,6 +14,9 @@
     },
     "require": {
         "php": ">=5.3.3",
-        "zendframework/zend-stdlib": "2.*"
+        "laminas/laminas-stdlib": "2.*"
```

Comment:
Let's try to update this dependency to the latest version if possible. Because of the version 2.x conflicts with other dependencies in Plesk.

Reply:
It's fine to do it separately.

Reply:
How is it possible, that the namespace was changed, but there are no changes in code, which uses the library? Is it used in fact?

Reply:
All laminas packages require `zend-framework-bridge` which maps Zend namespace onto Laminas.

Reply:
Looks like it uses only `Zend\Stdlib\ErrorHandler`.
I would drop this dependency, but probably not now.